lib/cadet/jobs/autograder/utilities.ex

```diff
@@ -0,0 +1,52 @@
+defmodule Cadet.Autograder.Utilities do
+  @moduledoc """
+  This module defines functions that support the autograder functionality.
+  """
+  use Cadet, :context
+
+  require Logger
+
+  import Ecto.Query
+
+  alias Cadet.Accounts.User
+  alias Cadet.Assessments.{Answer, Assessment, Submission}
+
+  def dispatch_programming_answer(answer, question) do
```

Comment:
JUST to be safer maybe do `def dispatch_programming_answer(answer = %Answer{}, question = %Question{}) do`